build.sbt

```diff
@@ -45,6 +45,13 @@ initialCommands in consoleQuick := ""
 Defaults.itSettings
 configs(IntegrationTest)
 
+// Fail builds if pull requests don't maintain test coverage. Bump this up as
+// coverage improves. Coverage is better if we include integration tests, but we
+// need to separate true end-to-end tests that require account credentials since
+// these are troublesome to run on CI.
+coverageMinimum := 60
+coverageFailOnMinimum := true
```

Comment:
This could potentially be left out because Codecov has similar native functionality, under the "Commit Status" settings menu item. It's not a minimum threshold though, it reports a failed check on GitHub if coverage decreases at all, so it's less lenient.

So, Travis build could be allowed to pass but coverage check GitHub integration could fail separately. The build failure could be useful with some other coverage service that doesn't have this feature, or if the overall minimum threshold setting is preferable. Open to your thoughts.